### PR TITLE
Fix bug when any functions haven't output nodes in Visual Script

### PR DESCRIPTION
**Pull request by wafemand**
https://github.com/godotengine/godot/issues/11811
Added difference between function which return value and function which don't return value . Now `ADDFUNC` with suffix `R` add function with return value. It was changed because `addfunc` think that function don't return value if it return `NIL`.

### CODE REVIEW

File: core/variant_call.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*************************************************************************/
+﻿/*************************************************************************/
```
Comment (akien-mga) [inline core/variant_call.cpp]: Looks like your IDE/code editor has added a BOM (Byte Order Mark) here, could you remove it?

File: core/variant_call.cpp [hunk 2]
```diff
@@ -54,8 +54,8 @@ struct _VariantCall {
 		Variant::Type return_type;
 
 		bool _const;
-#ifdef DEBUG_ENABLED
 		bool returns;
+#ifdef DEBUG_ENABLED
```
Comment (akien-mga) [inline core/variant_call.cpp]: Why that change? That `#ifdef` is useless now

**Comment by akien-mga** [comment]
Could you squash the commits together? See http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase

**Comment by wafemand** [comment]
Ok, I'll fix it soon.

**Comment by wafemand** [comment]
I suppose I made some strange actions(((

**Comment by wafemand** [comment]
@akien-mga 
I made a new pull request. This request may be deleted.
New pull request: https://github.com/godotengine/godot/pull/12382

**Pull request closed without merging**